sdk/python/kfp/_client.py

```diff
@@ -23,6 +23,7 @@
 import yaml
 import zipfile
 import datetime
+import requests
```

Comment:
Do you mind updating `sdk/requirements.txt` using `pip-compile` to make sure it reflects the latest dependency closure? Thanks

Reply:
Requests is actually not needed anymore. If I run `pip-compile --output-file=requirements.txt requirements.in` two things are removed: `importlib-metadata==1.5.0  # via jsonschema` and `zipp==1.2.0               # via importlib-metadata`. Given that requests is not necessary, it should unchanged. Should I leave it unchanged or the one with those 2 removed?

Reply:
Interesting... So I'm fine leave it as is. Thanks